gbpservice/nfp/test/unit/configurator/agent/lb_test_agent.py

```diff
@@ -4,537 +4,197 @@
+            'serialize': True,
+            'binding_key': '6350c0fd-07f8-46ff-b797-62acd23760de'}
+        self.arg_dict_pool_update = {
+            'context': self.fo.context,
+            'pool': self.fo._fake_pool_obj(),
+            'serialize': True,
+            'binding_key': '6350c0fd-07f8-46ff-b797-62acd23760de',
+            'old_pool': self.fo._fake_pool_obj()}
+        self.arg_dict_pool_delete = {
+            'context': self.fo.context,
+            'pool': self.fo._fake_pool_obj(),
+            'serialize': True,
+            'binding_key': '6350c0fd-07f8-46ff-b797-62acd23760de'}
+        self.arg_dict_member = {
+            'context': self.fo.context,
+            'member': self.fo._fake_member_obj()[0],
+            'serialize': True,
+            'binding_key': '6350c0fd-07f8-46ff-b797-62acd23760de'}
+        self.arg_dict_member_update = {
+            'context': self.fo.context,
+            'member': self.fo._fake_member_obj()[0],
+            'old_member': self.fo._fake_member_obj()[0],
+            'serialize': True,
+            'binding_key': '6350c0fd-07f8-46ff-b797-62acd23760de'}
+        self.arg_dict_health_monitor = {
+            'context': self.fo.context,
+            'health_monitor': self.fo._fake_hm_obj()[0],
+            'pool_id': self.fo._fake_pool_obj()['id'],
+            'serialize': True,
+            'binding_key': '6350c0fd-07f8-46ff-b797-62acd23760de'}
+        self.arg_dict_health_monitor_update = {
+            'context': self.fo.context,
+            'health_monitor': self.fo._fake_hm_obj()[0],
+            'old_health_monitor': self.fo._fake_hm_obj()[0],
+            'pool_id': self.fo._fake_pool_obj()['id'],
+            'serialize': True,
+            'binding_key': '6350c0fd-07f8-46ff-b797-62acd23760de'}
 
     @mock.patch(__name__ + '.FakeObjects.conf')
     @mock.patch(__name__ + '.FakeObjects.sc')
-    def _get_ConfiguratorRpcManager_object(self, sc, conf):
-        cm = cfgr.ConfiguratorModule(sc)
-        demuxer = demuxer_lib.ConfiguratorDemuxer()
-        rpc_mgr = cfgr.ConfiguratorRpcManager(sc, cm, conf, demuxer)
+    def _get_configurator_rpc_manager_object(self, sc, conf):
+        cm = configurator.ConfiguratorModule(sc)
+        dmx = demuxer.ConfiguratorDemuxer()
+        rpc_mgr = configurator.ConfiguratorRpcManager(sc, cm, conf, dmx)
         return sc, conf, rpc_mgr
 
-    def _get_GenericConfigRpcManager_object(self, conf, sc):
-        agent = gc.GenericConfigRpcManager(sc, conf)
+    def _get_lbaas_rpc_manager_object(self, conf, sc):
+        agent = lb.LBaaSRpcManager(sc, conf)
         return agent, sc
 
-    @mock.patch(__name__ + '.FakeObjects.nqueue')
-    @mock.patch(__name__ + '.FakeObjects.drivers')
-    def _get_GenericConfigEventHandler_object(
-            self, sc, rpcmgr, drivers, nqueue):
-        agent = gc.GenericConfigEventHandler(sc, drivers, rpcmgr, nqueue)
-        return agent
+    def _test_controller(self, operation):
```

Comment:
Please change function name to test_event_creation() as we are creating events. _test_controller() does not convey much info here.